app_flutter/lib/widgets/commit_box.dart

```diff
@@ -143,3 +162,42 @@ class CommitOverlayContents extends StatelessWidget {
     await launch(githubUrl);
   }
 }
+
+class Hyperlink extends StatefulWidget {
+  const Hyperlink({
+    Key key,
+    @required this.text,
+    this.onPressed,
+  })  : assert(text != null),
+        super(key: key);
+
+  final String text;
+  final VoidCallback onPressed;
+
+  @override
+  _HyperlinkState createState() => _HyperlinkState();
+}
+
+class _HyperlinkState extends State<Hyperlink> {
+  bool hover = false;
+
+  @override
+  Widget build(BuildContext context) {
+    return MouseRegion(
+      onEnter: (PointerEnterEvent _) => setState(() => hover = true),
```

Comment:
i thought we were going to create ourselves a Hover class that had two children, one when hovered and one when not, or a HoverBuilder class with a builder that takes a boolean, or something. Did we never get around to that? cc @gspencergoog  @dkwingsmt